### DISCUSSION TITLE
Discover the number of hosts under each network ID

### DISCUSSION THREAD

**Discussion by f18911238888** [sinamics/ztnet#218] [General]
### 🚀 Feature Summary

Discover the number of hosts under each network ID

### 📝 Detailed Description

Hello, how can ADMIN know the number of hosts under each network ID? Thank you.

### 💡 Willing to Contribute

None

**Comment by tinola** [comment]
I don't think you know the concept and how ZeroTier works...
Host will automatically be shown after successful joining the specified network - so there is no need to discover anything.

**Comment by f18911238888** [comment]
How do I limit the number of hosts per registered user

**Comment by sinamics** [comment]
Converted to discussion, as this is not related to an issue with ztnet. 

> How do I limit the number of hosts per registered user

Ztnet is just a wrapper arround zerotier. You cannot limit the amount of users in a ztnet controller, but you have the posibillitiy to delete and remove users. If you add a private network then all users must be autorized before they get access to the network peers.

**Comment by f18911238888** [comment]
For example, the group I created is ABC, and the registered user is abc@demo.com The validity period for this user is 30 days. Can you increase the number of peer hosts here (such as 10)? Thank you.

**Reply by f18911238888** [reply]
![image](https://github.com/sinamics/ztnet/assets/151608713/fc081314-957c-464f-ad45-ab94a400f146)

**Reply by sinamics** [reply]
This is not possible as the zerotier controller does not support such feature.
